language/examples/bqml/bqml_llm_examples.ipynb

```diff
@@ -0,0 +1,1114 @@
+{
```

Comment:
<p>Might be helpful to add some comments under "try" and "except" to explain what's happening in this block of code?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/127/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>updated - ta</p>